core/trino-web-ui/src/main/resources/webapp/assets/trino.css

```diff
@@ -426,6 +426,7 @@ pre {
 .query-header-timestamp {
     text-align: right;
     color: #b1b1b1;
+    margin-left: -15px;
```

Comment:
I noticed that .query also has the margin-left .. I wonder what happens if we remove that there instead..